bin/akula.rs

```diff
@@ -97,6 +97,10 @@ pub struct Opt {
     #[clap(long)]
     pub no_rpc: bool,
 
+    /// Enable API options
+    #[clap(long, min_values(1))]
+    pub enable_api: Vec<String>,
```

Comment:
Use HashSet here

Reply:
When changing this to a HashSet, I get a compilation error saying that 
```
   |
10 |     pub enable_api: HashSet<String>,
   |         ^^^^^^^^^^ the trait `FromStr` is not implemented for `HashSet<std::string::String>`
   |
```

I can make `enable_api` into a separate type that can implement the `FromStr` trait. I could also make the `enable_api` struct property a `Vec<String>` and then transform the value into a `HashSet` from there?

```rust
pub struct Opt {
...

    #[clap(long, min_values(1))]
    pub enable_api: Vec<String>,

...
}

#[allow(unreachable_code)]
fn main() -> anyhow::Result<()> {
...

    let args = Args::parse();
    let api_options: HashSet<String> = HashSet::from_iter(args.enable_api);
    
    //Check for each Id in the HashSet

...
}
```

Let me know if you have any guidance on what might work best.

Reply:
Keep Vec here, transform in main

Reply:
Sounds good, I will update this when I get back to my computer.

Reply:
I just updated my PR with these changes. Please let me know if it looks good or if you would like any further changes/adjustments.